test/scsb_rest_client.test.js

```diff
@@ -1,13 +1,55 @@
 /* eslint no-new:0*/
```

Comment:
Not really, I wanted to showcase the different approaches when testing promises. I will modify the codebase to use `should()` or `expect()`

Here is the documentation with `assert` -- https://github.com/domenic/chai-as-promised#assert-interface